app/backend/controllers/profile.js

```diff
@@ -213,3 +216,22 @@ module.exports.rejectRequest = async (request, response) => {
   })
 }
 
+/*
+  Get method for articles of user.
+*/
+module.exports.getArticles = async (request, response) => {
```

Comment:
The endpoint may be changed to singular and console logs can be removed too.

Reply:
Actually we will never call this function. So I will delete it.